rake_image/cloudbuild.yaml

```diff
@@ -0,0 +1,5 @@
+steps:
+  - name: gcr.io/cloud-builders/docker
+    args: ['build', 't', 'gcr.io/gcp-runtimes/rake', '.']
```

Comment:
hmm....yeah that's definitely what it should be, i'm wondering how this worked in the first place.